shared/src/commonMain/kotlin/co/nimblehq/mark/kmmic/data/service/api/ApiService.kt

```diff
@@ -60,6 +65,25 @@ internal class ApiService(requiresAuthentication: Boolean = false): KoinComponen
                                 BearerTokens(accessToken, refreshToken)
                             }
                         }
+
+                        sendWithoutRequest { request ->
+                            request.url.host == Url(BuildKonfig.BASE_URL).host
+                        }
+
+                        refreshTokens {
+                            print(this)
```

Comment:
We should remove the print call.

Reply:
@phongvhd93 updated at [b62ae69](https://github.com/markgravity/kmm-ic/pull/105/commits/b62ae69001d7118d72ac27657189d833b5ef7ffe)